### PR TITLE
{lib}[GCCcore/10.2.0] FORD v6.1.1

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FORD/FORD-6.1.1-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+dependencies = [
+    ('Python', '3.8.6'),
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FORD/FORD-6.1.1-GCCcore-10.2.0.eb:19]: @SimonPinches please merge these into the first `dependencies`

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/FORD/FORD-6.1.1-GCCcore-10.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/f/FORD/FORD-6.1.1-GCCcore-10.2.0.eb:15]: @verdurin Oops! Not quite sure how I managed that but all fixed now ;-)

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a9d1bc597e4ce337b6149e35079b448e for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @SimonPinches!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (2 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/11bc2687e6530e53fcf5aa627613de44 for a full test report.

**Pull request merged**